Cargo.toml

```diff
@@ -13,50 +13,45 @@ repository = "https://github.com/stackabletech/secret-operator"
 [workspace.dependencies]
 anyhow = "1.0"
 async-trait = "0.1"
-bindgen = "0.66"
-built = { version = "0.6", features = ["chrono", "git2"] }
-byteorder = "1.4"
-clap = "4.3"
+bindgen = "0.69.4"
+built = { version = "0.7", features = ["chrono", "git2"] }
+byteorder = "1.5.0"
+clap = "4.5.4"
 futures = { version = "0.3", features = ["compat"] }
-h2 = "=0.3.18" # We need to keep this in sync with our patched h2 build
-ldap3 = { version = "0.11", default-features = false, features = [
-  "gssapi",
-  "tls",
-] }
+h2 = "=0.3.26" # We need to keep this in sync with our patched h2 build
```

Comment:
I don't know if this will cause problems to be honest, when your confident enough merge it :)

Reply:
Can you elaborate?
I don't understand enough about this h2 patch, and if it silently breaks things, or if it just doesn't compile.

Reply:
I don't know this details neither :) I just know someone broke something in the past, hence we added this comment.
I personally would revert this line and let Nat take care of the bump

Reply:
Just waiting on some confirmation from @nightkr, I will either leave as is for now, or roll it back and wait for https://github.com/stackabletech/h2/pull/3

Reply:
That is now merged. The plan here is to keep h2 at `=0.3.26` (in this PR), but revert the patch branch back to `feature/grpc-uds`. I aim to do this tonight, but wrote this here in case someone needs it done earlier.

Reply:
Fixed in a42f10b